WcaOnRails/app/models/wcif.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Wcif < ApplicationRecord
+  belongs_to :competition
+
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
+  def json
+    json_db = JSON.parse(super || "{}")
+    json_db.merge(
+      "formatVersion" => "1.0",
+      "id" => competition.id,
+    )
+  end
+
+  def json=(new_json)
+    json = self.json.merge(new_json)
+    super(JSON.generate(json))
+  end
+
+  def json_hash
```

Comment:
Normally I'd agree with "let's keep it simple for now and implement it later", but with APIs, my thinking gets all convoluted. Agreed that we should table this discussion until we figure out the base API first.